docs/learning/presentation/Content/PropertyCategorization.md

```diff
@@ -20,19 +20,23 @@ If a property has no direct category, it is displayed as if it belonged to the p
 
 Categories group properties so they're easier for users to find and manage. However, it's often necessary to group already categorized properties into some other group, for example 'favorites'. This is where nested categorization comes into play.
 
-Generally, since a property grid requests properties for the current selection, we put all properties into the *Selected Item(s)* category:
+Generally, since a property grid requests properties for the current selection, we put all properties into the default *Selected Item(s)* category:
 
 ![Selected Item(s)' Category](./media/property-grid-selected-items-category.png)
 
+The default category can be overriden using [DefaultPropertyCategoryOverride rule](./DefaultPropertyCategoryOverride.md).
+
 If properties have [direct categories of their own](#direct-property-categories), their categories appear nested under this top-level category, e.g.:
 
 ![Nested Categories](./media/property-grid-nested-categories.png)
 
-Additionally, when presentation rules specify [related properties](./RelatedPropertiesSpecification.md), it is possible to create new top-level categories. See the [next chapter](#controlling-category-nesting-of-related-properties) for more details.
+Additionally, when presentation rules specify [related properties](./RelatedPropertiesSpecification.md), it is possible to create new top-level categories. See [this topic](#controlling-category-nesting-of-related-properties) for more details.
+
+Finally, category nesting can be achieved using [property category overrides](./PropertyCategorySpecification.md) - see [this topic](#controlling-category-nesting-through-property-category-overrides) for more details.

```

Comment:
Phrasing could be improved to disclose the hyperlink targets, as it's now unclear whether they'll navigate user to another page. Try replacing the word "topic", e.g. "See \[the following section\](#controlling-category-nesting-of-related-properties) for more details."

Reply:
thanks, done